website/source/docs/http/sys-mounts.html.md

```diff
@@ -31,12 +36,54 @@ description: |-
     {
       "aws": {
         "type": "aws",
-        "description": "AWS keys"
+        "description": "AWS keys",
+        "config": {
+          "default_lease_ttl": 0,
+          "max_lease_ttl": 0
+        }
       },
 
       "sys": {
         "type": "system",
-        "description": "system endpoint"
+        "description": "system endpoint",
+        "config": {
+          "default_lease_ttl": 0,
+          "max_lease_ttl": 0
+        }
+      }
+    }
+    ```
+
+  </dd>
+</dl>
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    List the given secret backends configuration. `default_lease_ttl`
+    or `max_lease_ttl` values of `0` mean that the system
+    defaults are used by this backend.
+  </dd>
+
+  <dt>Method</dt>
+  <dd>GET</dd>
+
+  <dt>URL</dt>
+  <dd>`/sys/mounts/<mount point>/tune`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
+    None
+  </dd>
+
+  <dt>Returns</dt>
+  <dd>
+
+    ```javascript
+    {
+      "config": {
```

Comment:
I did that purely to keep it consistent with mounts, but I guess that doesn't make sense since what it returns (the systemview values instead of the mount entry values) is different. So I'll change that.